src/main/java/com/kmush12/CurrencyConverter/exchange/ExchangeRate.java

```diff
@@ -1,11 +1,62 @@
 package com.kmush12.CurrencyConverter.exchange;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
 
 import java.math.BigDecimal;
 import java.util.HashMap;
+import java.util.Objects;
+
+@Document
+public final class ExchangeRate {
+    @Id
+    private String id;
+    private final String originCurrency;
+    @JsonProperty("rates")
```

Comment:
Ta adnotacja jest na pewno potrzebna skoro nazwa pola jest taka sama?
Poza tym nie powinniśmy mieszać adnotacji bazodanowych (mongo) z tymi do wyświetlania danych (jackson). Najlepiej jakby to były dwie osobne klasy, chociaż wiem, że w tym przypadku jest to trochę na wyrost -- na razie niech zostanie jedna ale zobacz czy da się pozbyć tych adnotacji z jacksona.